corehq/form_processor/models.py

```diff
@@ -499,9 +498,7 @@ def form_data(self):
             form_json = convert_xform_to_json(xml)
         except XMLSyntaxError:
             return {}
-        # we can assume all sql domains are new timezone domains
-        with force_phone_timezones_should_be_processed():
```

Comment:
Ah okay this is the line that makes me confident that this PR has no consequences. It looks like we had never explicitly set the tzmigration to complete, but that the SQL forms implementation ignores its value anyway.